src/brs/Burst.java

```diff
@@ -120,21 +118,14 @@ public static void main(String[] args) {
     init();
   }
 
-  private static void validateVersionNotDev(PropertyService propertyService) {
+  private static boolean validateVersionNotDev(PropertyService propertyService) {
```

Comment:
Invalid returns false but do not shutdown so BurstGUI can still show what went wrong.